install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh 
+
+#Author: Marina Schmidt
+#
+#This script is used to setup the bash environmnent variables for the use of mappot_fitacf 
+#
+export PATH=`pwd`:$PATH
```

Comment:
input paths and output paths should be separate variables, and would prefer to have them required command line flags. You could also use this within mappot_fitacf to find its own path (with key and extension files also required to be in its own path) : https://stackoverflow.com/questions/39340169/dir-cd-dirname-bash-source0-pwd-how-does-that-work